Sources/Fluent/QueryBuilder/QueryBuilder+CRUD.swift

```diff
@@ -52,6 +52,9 @@ extension QueryBuilder where Result: Model, Result.Database == Database {
     public func update<T>(_ field: KeyPath<Result, T>, to value: T) -> Self where T: Encodable {
         Database.queryActionApply(Database.queryActionUpdate, to: &query)
         Database.queryDataSet(Database.queryField(.keyPath(field)), to: value, on: &query)
+        if Result.updatedAtKey != nil {
+            Database.queryDataSet(Database.queryField(.keyPath(\Result.fluentUpdatedAt)), to: Date(), on: &query)
```

Comment:
I think `.keyPath(\Result.fluentUpdatedAt)` should be `.keyPath(Result.updatedAtKey)` (using `if let` to unwrap the optional).

Reply:
Sounds good! I'll submit a test soon. I just realized `FluentBenchmark` exists and I see now how tests are being done.